samples/native/test/main.c

```diff
@@ -23,37 +23,36 @@
 
 /* Private Data. */
 /* All static data definitions appear here. */
-RequestServiceInfo rsi = { 
-  .ExecutionType  = "native",
-  .ExeCmd         = "ls"
+RequestServiceInfo rsi = {
+	.ExecutionType  = "native",
+	.ExeCmd         = "ls"
```

Comment:
@tdrozdovsky Same question about my recent review in terms of code indentation.

Reply:
@MoonkiHong Sorry for the misunderstanding. I'm a bit confused myself.
1. As far as I know, the `gofmt` is designed for the Go language (maybe I'm wrong), and this file is in C
2. Since Go is used for `tab` symbol for indentation, I applied it to make it look as much like formatting for Go

I am ready to do as you say. Sorry again

Reply:
@tdrozdovsky Okay, well understood. Then your current code indentation is based on 6 spaces, instead of 4 spaces. Please verify it again and let me know. (Previously I have recommend you a 4 spaces)

Reply:
Done)))